File: Models/ViewModels/UserVM.cs
namespace GoWheels_WebAPI.Models.ViewModels
{
    public class UserVM
    {
        public string? Id { get; set; }
        public string? Name { get; set; }
        public string? License { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string? CIC { get; set; }
        public string? Image { get; set; }
        public string? Longitude { get; set; }
        public string? Latitude { get; set; }
        public bool LockoutEnabled { get; set; }
        public DateTimeOffset LockoutEnd { get; set; }
        public int? ReportPoint {  get; set; }
        public DateTime? Birthday { get; set; }
        public IList<string> Roles { get; set; } = new List<string>();
    }
}

File: FE/src/Model/UserVM.js
import DriverVM from "./DriverVM";
export default class UserVM {
  constructor(data = {}) {
    this.id = data.id || null;
    this.name = data.name || "";
    this.createDate = data.createDate || null;
    this.email = data.email || "";
    this.phoneNumber = data.phoneNumber || "";
    this.lockoutEnd = data.lockoutEnd || null;
    this.lockoutEnabled = data.lockoutEnabled || false;
    this.isSubmitDriver = data.isSubmitDriver || false;
    this.cic = data.cic || "";
    (this.license = data.license || ""),
      (this.reportPoint = data.reportPoint || 0),
      (this.image = data.image || ""); // Đường dẫn hoặc URL của ảnh icon từ API
    (this.birthday = data.birthday || null), (this.roles = data.roles || []);
  }
}


